src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -663,7 +663,7 @@ extern "C" {
 /**
  * Default minimum number of threads in the threadpool for the SDK
  */
-#define THREADPOOL_MIN_THREADS 3
+#define THREADPOOL_MIN_THREADS 1
```

Comment:
It doesn't defeat the purpose. The main reason we first added threadpool was a means of re-using threads **and** queuing tasks, as before we would just ignore the task if creating the thread failed.

However we also want to show how lean the SDK can be, and reducing to the minimum number of handlers is aligned with that goal.

This is meant to be something that can be tuned for any given use case.

Reply:
Hmm. Ok. Since the value is controllable via exports, I dont have a strong opinion about the value. There is a line in the README: 

`The threadpool is enabled by default, and starts with 5 threads that it can increase up to 10 if all are actively in use`

Can we just change this to reflect the correct value?